### PR TITLE
Modernize with Embulk v0.10.25

### PR DESCRIPTION
**Pull request by huylenq**
- Upgrade `embulk-*` to `v0.10.25`.
- Apply `embulk-util-config` (`v0.2.1`).
- Upgrade Gradle to `6.6.1`.
- Apply `org.embulk.embulk-plugins` Gradle plugin.
- Miscellaneous code cleanup.

### CODE REVIEW

File: src/main/java/org/embulk/output/mailchimp/helper/MailChimpHelper.java [hunk 1]
```diff
@@ -83,7 +83,7 @@ public String apply(@Nullable JsonNode input)
             }
```
Comment (xuantuan58) [inline src/main/java/org/embulk/output/mailchimp/helper/MailChimpHelper.java]: the method `MailchimpHeloper#extractMemberStatus` is test helper method, consider moves it to the test package

Comment (xuantuan58) [reply src/main/java/org/embulk/output/mailchimp/helper/MailChimpHelper.java]: Also please moves the `containsCaseInsensitive `

Comment (vietnguyen-td) [reply src/main/java/org/embulk/output/mailchimp/helper/MailChimpHelper.java]: updated

**Review by xuantuan58 — APPROVED** [review]
LGTM

**Comment by vietnguyen-td** [comment]
hi @ylzen , could you help to review Data Exfiltration?

**Review by ylzen — APPROVED** [review]
Reviewed for Data Exfiltrations.

**Pull request merged**